packages/node_modules/samples/browser-plugin-meetings/index.html

```diff
@@ -346,6 +346,7 @@ <h2>Then click:</h2>
       <canvas id="myChart2"></canvas>
     </div>
         <h2>Extras</h2>
+        <button id="upload-logs">uploadLogs</button>
```

Comment:
Or you could add `type="button"` to not send the form, instead of `event.preventDefault();`
```suggestion
        <button id="upload-logs" type="button">uploadLogs</button>
```

Reply:
`button` and `type=button` are the same thing. `type=button` is nice if we wanted to use a `span` or `div` here.

Reply:
@mwegman  In this case is true because we are not inside a `<form>` element, otherwise, the form would submit if we don't specify `type="button"`. @adamweeks can you juts pass the `event` as parameter in order to use it in `event.preventDefault()` then? I'm pretty sure that will break the code since `event` is undefined.